spec/models/cheatsheet_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+RSpec.describe Cheatsheet, type: :model do
+
+  before :context do
```

Comment:
I'm not familiar with the `before :context`. What's that doing?

Reply:
It runs the block of code, once, before everything in the group.

example:
```ruby
RSpec.describe Thing do
  before(:example) do
    @thing = Thing.new
  end

  describe "initialized in before(:example)" do
    it "has 0 widgets" do
      expect(@thing.widgets.count).to eq(0)
    end

    it "can accept new widgets" do
      @thing.widgets << Object.new
    end

    it "does not share state across examples" do
      expect(@thing.widgets.count).to eq(0)
    end
  end
end
```

Reply:
Ahh ... I see.